walkthroughs/3a-low-code-api-development/walkthrough.adoc

```diff
@@ -56,18 +63,25 @@ If you already have admin access to a Slack workspace, skip this procedure.
 
 . Enter your email address and click *Next*.
 
-. Enter a name for your workspace and click *Next*.
+. Enter the following name for your workspace and click *Next*:
++
+----
+low-code-workspace
```

Comment:
Can't use a preset name. Each person must use something unique.

Reply:
Note, this is because this determines the Slack workspace URL, e.g `low-code-workspace.slack.com` can only exist once.

Reply:
what I found was that if you entered a non-unique name, slack gives you <name-uid> and since we can't anticipate that the user would think of a unique name, might as well use a preset name and then ask them to note the resulting url.

Reply:
Interesting. I didn't know that 👍